runtime/CMakeLists.txt

```diff
@@ -215,12 +215,14 @@ add_subdirectory("${J9VM_OMR_DIR}" omr)
 # Note: this has to happen after we add_subdirectory omr
 
 # The string we append to the name of shared libs
-set(versionStr "${J9VM_VERSION_MAJOR}${J9VM_VERSION_MINOR}")
+#set(versionStr "${J9VM_VERSION_MAJOR}${J9VM_VERSION_MINOR}")
+set(versionStr "29")
```

Comment:
Why hard code? This is not ideal.

Reply:
I agree, however it seems that we are doing that in the makefiles, so I tried to mirror that.